docs/RELEASE_NOTES.md

```diff
@@ -26,6 +26,14 @@
 
 7. Add Migration guide section which specifies explicitly if there are breaking changes and how to tackle them.
 -->
+### 4.0.0 - Mar xx, 2021
+[Commits](https://github.com/commercetools/commercetools-sync-java/compare/3.2.0...4.0.0) |
+[Javadoc](https://commercetools.github.io/commercetools-sync-java/v/4.0.0/) | 
+[Jar](https://bintray.com/commercetools/maven/commercetools-sync-java/4.0.0)
+
+- 🚧 **Breaking Changes** (1)
+    - **Product Sync**: `PriceDraft.getCustomerGroup()` is changed from `Reference<CustomerGroup>` to `ResourceIdentifier<CustomerGroup>`, so as a library user you don't need to provide a key field in the id field of the Reference. (Now API and JVM SDK support `ResourceIdentifiers` and it supports id or key as a field). [#676](https://github.com/commercetools/commercetools-sync-java/pull/676)
+
```

Comment:
Could you please add dependency updates JVM-sdk here?

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/676/commits/28abccf5711bfdda1ddb910a29c7671fb5cddcd8

Reply:
would be better from `1.57.0` because there were no changes that we used and document after this version: https://commercetools.slack.com/archives/CCJALDJ9F/p1612368786005400

Reply:
https://github.com/commercetools/commercetools-sync-java/pull/676/commits/e7670fcc325a51ac0217d8b16aa762baec62b38e